client/src/dist/index.html

```diff
@@ -10,6 +10,7 @@
 </head>
 
 <body>
+  <div id="root"></div>
```

Comment:
This isn't needed. We're creating the root div in src/index.js already. 
Otherwise everything else looks great.